### PR TITLE
ssh: simplify page

### PR DESCRIPTION
**Pull request by agnivade**
- Combined the last 2 commands into a single format that can be expanded upon
by showing the -o option, since both can be passed as options.

### CODE REVIEW

**Comment by agnivade** [comment]
Fixes #970

**Comment by sbrl** [comment]
I've just look a little into SSH Agent Forwarding, and it looks awesome! I've never heard of it before, but I really wish I had. it should totally be in tldr-pages :+1:

**Comment by agnivade** [comment]
ssh has a lot of hidden features which can blow your mind. https://vimeo.com/54505525 is highly recommended

File: pages/common/ssh.md [hunk 1]
```diff
@@ -27,6 +27,6 @@
 
 `ssh -L {{9999}}:slashdot.org:80 {{username}}@{{remote_host}}`
 
-- SSH enable agent forward:
+- Forward the authentication information to the remote machine by passing an option (For full list of possible options, try `man ssh_config`):
```
Comment (waldyrious) [inline pages/common/ssh.md]: I'd shorten this description a bit. How about "Enable the option to forward the authentication information to the remote machine (see `man ssh_config` for available options):"? This has the benefit of following the same order as the elements of the command.

**Comment by waldyrious** [comment]
👍 LGTM

**Comment by sbrl** [comment]
LGTM! :+1: (since LGTM doesn't seem to pick up on my previous comment)

**Comment by waldyrious** [comment]
@sbrl the LGTM service only recognizes approvals by project collaborators/maintainers. I'm not sure it can be configured to accept any user's approval, but I'd favor setting it that way if possible.

**Comment by sbrl** [comment]
@waldyrious Ah right! I didn't know that.

**Comment by agnivade** [comment]
:shipit:

**Pull request merged**